src/sty/arrow.sty

```diff
@@ -1,20 +1,27 @@
-Set x {
-    shape = Circle { }
-    objective outside(x.shape.label, x.shape)
-    arr = Arrow {
-        start = x.shape.label
-        end   = x
-        text = None
-    }
+Vector v {
+    v.shape = Parallelogram {
+    x = 10.0
+    y = 10.0
+    lengthX = 200.0
+    lengthY = 200.0
+    angle = 130
+    rotation = 20.0
+   }
 }
 
-NoIntersect x y {
-    constraint nonOverlapping(x.shape, y.shape)
-}
+Scalar s {
+    v.shape = Arc {
```

Comment:
The parallelogram is draggable in the frontend but the angle mark doesn't seem to be. Can you make the latter draggable?

Reply:
See my explanation below